### PR TITLE
[CO-2860] Updated /users/export/ids documentation to reflect the new rate limit / request param

### PR DESCRIPTION
**Pull request by shihanxiong**
Screenshot 2024-08-27 at 4 01 15 PM

### Summary

`/users/export/ids` will have following changes starting Aug 22, 2024
- 250 / min rate limit
- `fields_to_export` is no longer optional

### Related PRs, issues, or features (optional)

- After Aug 22, 2024

### Feature release date (optional)

- N/A

### Contributor checklist

- [x] I confirm that my PR meets the following:
    - I signed the [Contribution License Agreement (CLA)](https://www.braze.com/docs/cla).
    - My style and voice follows the [Braze Style Guide](https://docs.google.com/document/u/2/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub).
    - My content contains correct spelling and grammar.
    - All links are working correctly.
    - If I renamed or moved a file or directory, I set up [URL redirects](https://www.braze.com/docs/contributing/content_management/redirecting_urls) for each file.
    - If I updated or replaced an image, I did not remove the original image file from the repository. (For more information, see [Updating an image](https://www.braze.com/docs/contributing/content_management/images/#updating-an-image)).
    - If my PR is related to a paid SKU, third party, SMS, AI, or privacy, I have received written approval from Braze Legal.

### Submitting for review

If your PR meets the above requirements, select **Ready for review**, then add a reviewer:
- **Non-Braze contributors:** Add `braze-inc/docs-team` as the reviewer.
- **Braze employees:** Have any relevant subject matter experts (like engineers or product managers) review your work first, then add the [tech writer assigned to your specific vertical](https://confluence.braze.com/pages/viewpage.action?pageId=75039896). If you're not sure which tech writer to add, you can add `braze-inc/docs-team` instead. 

Thanks for contributing! We look forward to reading your work.

### CODE REVIEW

File: _docs/_api/endpoints/export/user_data/post_users_identifier.md [hunk 1]
```diff
@@ -43,21 +43,25 @@ Authorization: Bearer YOUR-REST-API-KEY
   "braze_id": (optional, string) Braze identifier for a particular user,
   "email_address": (optional, string) Email address of user,
   "phone": (optional, string) Phone number of user,
-  "fields_to_export": (optional, array of strings) Name of user data fields to export. Defaults to all if not provided
+  "fields_to_export": (required, array of strings) Name of user data fields to export. Defaults to all if not provided
```
Comment (davehensley) [inline _docs/_api/endpoints/export/user_data/post_users_identifier.md:46]: Probably want to get rid of "Defaults to all...", right?

**Review by davehensley — APPROVED** [review]

File: _includes/rate_limits.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ See our page on [API rate limits]({{site.baseurl}}/api/api_limits/) for details,
 <!---/users/export/ids-->
 
 {% elsif include.endpoint == "users export ids" %}
-For customers who onboarded with Braze on or after August 16, 2021, we apply a rate limit of 2,500 requests per minute to this endpoint, as documented in [API rate limits]({{site.baseurl}}/api/api_limits/).
+For customers who onboarded with Braze on or after August 22, 2024, we apply a rate limit of 250 requests per minute to this endpoint, as documented in [API rate limits]({{site.baseurl}}/api/api_limits/).
```
Comment (lydia-xie) [inline _includes/rate_limits.md:43]:
```suggestion
This endpoint has a rate limit of 250 requests per minute, as documented in [API rate limits]({{site.baseurl}}/api/api_limits/).
```

File: _docs/_api/endpoints/export/user_data/post_users_identifier.md (hunk 1 above)
Comment (shihanxiong) [reply _docs/_api/endpoints/export/user_data/post_users_identifier.md]: Yea good point,  updated

File: _docs/_api/endpoints/export/user_data/post_users_identifier.md [hunk 3]
```diff
@@ -43,21 +43,25 @@ Authorization: Bearer YOUR-REST-API-KEY
   "braze_id": (optional, string) Braze identifier for a particular user,
   "email_address": (optional, string) Email address of user,
   "phone": (optional, string) Phone number of user,
-  "fields_to_export": (optional, array of strings) Name of user data fields to export. Defaults to all if not provided
+  "fields_to_export": (required, array of strings) Name of user data fields to export.
 }
 ```
 
+{% alert note %}
+For customers who onboarded with Braze on or after August 22, 2024. Request parameter `fields_to_export` will be required.
```
Comment (lydia-xie) [inline _docs/_api/endpoints/export/user_data/post_users_identifier.md:51]:
```suggestion
For customers who have onboarded with Braze on or after August 22, 2024, the request parameter `fields_to_export` is required.
```

File: _docs/_api/endpoints/export/user_data/post_users_identifier.md [hunk 4]
```diff
@@ -43,21 +43,25 @@ Authorization: Bearer YOUR-REST-API-KEY
   "braze_id": (optional, string) Braze identifier for a particular user,
   "email_address": (optional, string) Email address of user,
   "phone": (optional, string) Phone number of user,
-  "fields_to_export": (optional, array of strings) Name of user data fields to export. Defaults to all if not provided
+  "fields_to_export": (required, array of strings) Name of user data fields to export.
```
Comment (lydia-xie) [inline _docs/_api/endpoints/export/user_data/post_users_identifier.md:46]:
```suggestion
  "fields_to_export": (required, array of strings) Name of user data fields to export
```

**Pull request merged**